tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     EnvSetupScript: setup_env.bat
     buildArch: x64
     setVcvars: true    
-  timeoutInMinutes: 90
+  timeoutInMinutes: 120
```

Comment:
Need to increase the timeout to make CI build pass. Now all the windows CIs are timeout.